CONFIG.md

```diff
@@ -160,6 +160,11 @@ If you're not using the `shared-home` volume, then you can declare your own cust
 volume in the "Additional Volumes" section below, then declare the libraries as above
 (but with your custom volume name).
 
+## Request body size
```

Comment:
I think it is worth mentioning that this is specific to NGINX ingress, for example with:

`## Request body size (NGINX ingress only)`

Reply:
And its actually called `ingress-nginx` - we should include the URL to this annotation (`proxy-body-size`) https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#custom-max-body-size